researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -531,19 +534,7 @@ public Task loadTask(Context context, SchedulesAndTasksModel.TaskScheduleModel t
     public void uploadTaskResult(Context context, TaskResult taskResult) {
         // TODO: Update/Create TaskNotificationService
 
-        boolean isActivity = false;
-        if (taskResult.getTaskDetails().containsKey(ActiveTaskActivity.ACTIVITY_TASK_RESULT_KEY)) {
```

Comment:
Can you delete all references to task details then?  Seems like it is useless now.